tests/components/bond/test_config_flow.py

```diff
@@ -572,6 +575,7 @@ async def test_zeroconf_already_configured_no_reload_same_host(
             ),
         )
         await hass.async_block_till_done()
+        mock_async_schedule_reload.assert_called()
```

Comment:
It's been a while but I think the intent here was to verify it did not actually reload when nothing changed

Reply:
fixed